core/common/src/main/java/alluxio/grpc/GrpcUtils.java

```diff
@@ -106,8 +106,8 @@ public static AccessControlList fromProto(PAcl pAcl) {
       acl = new AccessControlList();
     }
 
-    acl.setOwningUser(pAcl.getOwner());
-    acl.setOwningGroup(pAcl.getOwningGroup());
+    acl.setOwningUser(pAcl.getOwner().intern());
```

Comment:
I'm trying to guard all callers of 
```
ACL.setOwningUser()
ACL.setOwningGroup()
MutableInodeFile.setOwningUser()
MutableInodeFile.setOwningGroup()
MutableInodeDirectory.setOwningUser()
MutableInodeDirectory.setOwningGroup()
```
However I'm not 100% percent sure those are the best places. Maybe intern() when we create the Subject in UserState or getting from that?